### ISSUE TITLE
Stacks are getting lost after plasma restart / relog

### ISSUE THREAD

**Issue by ciolansteen** [naraesk/plasma-docker#12]
It may be an issue with the installation but after I add a stack, each time I restart plasma or relog /reboot, all the containers disappear from the widgets's list.  

I tried installing the aur package and also building the package as documented. The behaivour is the same in both cases.

**Comment by naraesk** [comment]
Strange, I never experienced this behavior. But besides that, the widget is working as expected?

**Comment by ciolansteen** [comment]
Not really. This behaviour is the same since first time I tested this widget few months ago. I'm on archlinux and I tried both the aur package and building it from here (both old and clean os installation on a new laptop).
Also, the icons are not rendered correctly (see attached image).
![Screenshot_20210217_114147](https://user-images.githubusercontent.com/8395752/108185763-550b7c00-7115-11eb-9f0c-d4d495e0cf20.png)

**Comment by naraesk** [comment]
Thanks for reporting, will have a look at it.

**Comment by ciolansteen** [comment]
If you need any more info / testing / etc, i'll be more than glad to help because this plugin is an amazing idea.

**Comment by ciolansteen** [comment]
Another issue would be that only .yml files are recognized as docker compose files and .yaml are ignored but loaded fine after the filter is being removed.
![Screenshot_20210217_115040](https://user-images.githubusercontent.com/8395752/108187052-be3fbf00-7116-11eb-8cda-1ab5c4201d00.png)
![Screenshot_20210217_115315](https://user-images.githubusercontent.com/8395752/108187056-bed85580-7116-11eb-9b88-4e6e0834f592.png)
